pkg/scheduler/core/generic_scheduler.go

```diff
@@ -315,8 +314,7 @@ func (g *genericScheduler) findNodesThatPassExtenders(pod *v1.Pod, feasibleNodes
 		feasibleList, failedMap, err := extender.Filter(pod, feasibleNodes)
 		if err != nil {
 			if extender.IsIgnorable() {
-				klog.Warningf("Skipping extender %v as it returned error %v and has ignorable flag set",
-					extender, err)
+				klog.InfoS("Skipping extender as it returned error and has ignorable flag set", "extender", extender, "error", err)
```

Comment:
migration guide suggests the key `err` https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#using-errors

Reply:
Using ErrorS is more appropriate, I'll fix it.

Reply:
It's not. This is an expected error

Reply:
Do you mean adding a level to InfoS (klog.V(1).Infos) for the expected error?
Simplified mapping between functions:
klog.Warning, klog.Warningf, klog.Warningln, klog.WarningDepth -> klog.InfoS
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#change-log-functions-to-structured-equivalent

Reply:
I guess @alculquicondor  wants something like this, 
`klog.InfoS("Skipping extender as it returned error and has ignorable flag set", "extender", extender, "err", err)`

where `err` is the key instead of `error`.

Reply:
Thanks for reminding me. It's done. @chendave